x/subscription/keeper/subscription_test.go

```diff
@@ -327,63 +347,107 @@ func TestMonthlyRechargeCU(t *testing.T) {
 	projectKeeper := ts.keepers.Projects
 
 	account := common.CreateNewAccount(ts._ctx, *ts.keepers, 10000)
+	anotherAccount := common.CreateNewAccount(ts._ctx, *ts.keepers, 10000)
 	creator := account.Addr.String()
 
-	err := keeper.CreateSubscription(ts.ctx, creator, creator, ts.plans[0].Index, 2, "")
+	err := keeper.CreateSubscription(ts.ctx, creator, creator, ts.plans[0].Index, 3, "")
 	require.Nil(t, err)
 
-	block1 := uint64(ts.ctx.BlockHeight())
+	// add another project under the subcscription
+	projectData := projectstypes.ProjectData{
```

Comment:
can you reuse or enhance `prepareProjectsData()` here?

Reply:
there is no such function?